test-crates/duchess-java-tests/tests/ui/duplicate_implements.stderr

```diff
@@ -1,5 +1,5 @@
 error: error in class `log.Builder`: duplicate reference in 'implements' to 'log.BuildStep'
- --> $DIR/duplicate_implements.rs:4:5
```

Comment:
it's ok to replace `$DIR` with `test/ui` here?

Reply:
yea, this is a leftover from the last ui_test version bump